### DISCUSSION TITLE
App layout is partially broken when device' screen zoom/font size settings are changed

### DISCUSSION THREAD

**Discussion by pavloburykh** [status-im/status-legacy#16663] [General]
As has been pointed out thanx to @ibrkhalil, app layout doesn't fit well after changing device' screen zoom/font style settings. This results in poor user experience for those users who are using their device with changed screen zoom/font size setting due to sight issues. 

**Steps:** 
1. Change screen zoom/font size settings: set everything to max value for better result observing
2. Open Status app
3. Go through different screen
4. Pay attention at layout

**Actual result:** layout looks pretty unusable broken, some elements (like bottom tabs) are unvisible. 

https://user-images.githubusercontent.com/97245802/208696747-e3f32812-3a1a-4106-8402-0e1a4179affe.mp4

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by ibrkhalil** [comment]
We can scale text to match what's selected in settings like [here](https://spin.atomicobject.com/2021/11/12/3-ways-scale-react-native-app/#:~:text=The%20fontScale%20can%20range%20from,font%20scale%20of%20the%20device.)

**Comment by smohamedjavid** [comment]
> We can scale text to match what's selected in settings like [here](https://spin.atomicobject.com/2021/11/12/3-ways-scale-react-native-app/#:~:text=The%20fontScale%20can%20range%20from,font%20scale%20of%20the%20device.)



Or we can lock the text size and don't allow it to scale in respect to accessibility settings If we don't support scaling of UI in our app.

We just need to set [`allowFontScaling`](https://reactnative.dev/docs/text#allowfontscaling) props to `false` on text components.

**Comment by VolodLytvynenko** [comment]
ISSUE 2: All options are not fully shown in the opened pop after long tap on the message on the pinned messages list
**Steps to reproduce:**
1. Go to 1-1 chat
2. Send the message
3. Pinn the message
4. Open list of pinned messages
5. Long tap on the any message

**Actual result:**
Some options are out of the screen

https://user-images.githubusercontent.com/52490791/211608865-1ec7aa04-f8b1-4167-9988-30751cd262b1.mp4

**Expected result:**
All options are shown
![image](https://user-images.githubusercontent.com/52490791/211609185-859f5b3f-461a-4de1-8865-6405501600ee.png)


**Env:** Iphone 11 pro; Android studio: resolution1080x2400 dpi 420,

**Comment by cammellos** [comment]
I believe this requires ux input, moving to discussion
